plotly_resampler/aggregation/aggregators.py

```diff
@@ -273,18 +248,13 @@ def _arg_downsample(
     ) -> np.ndarray:
         size_threshold = 10_000_000
         ratio_threshold = 100
-
-        # TODO -> test this with a move of the .so file
-        if LTTB_core.__name__ == "LTTB_core_py":
-            size_threshold = 1_000_000
-
+        downsampler = LTTBDownsampler()
         if y.shape[0] > size_threshold and y.shape[0] / n_out > ratio_threshold:
-            # TODO: add argument for 30 when the paper is published
-            idxs = self.minmax._arg_downsample(x, y, n_out * 30)
-            y = y[idxs]
-            if x is not None:
-                x = x[idxs]
-        return self.lttb._arg_downsample(x, y, n_out)
+            downsampler = MinMaxLTTBDownsampler()
+
+        if x is None:
+            return downsampler.downsample(y, n_out=n_out)
+        return downsampler.downsample(x, y, n_out=n_out)
```

Comment:
What do we do with argument such as parallel?

Reply:
kwargs?

Reply:
maybe `**downsample_kwargs` then?